src/main/authentication/Security/Oauth/LogoutHandler.php

```diff
@@ -61,6 +56,8 @@ public function logout(Request $request, Response $response, TokenInterface $tok
                 if ($resourceOwner) {
                     $resourceOwner->revokeToken($resourceOwnerToken['token']);
                 }
+
+                $this->eventDispatcher->dispatch(SecurityEvents::USER_LOGOUT, UserLogoutEvent::class, [$token->getUser()]);
```

Comment:
can be moved after the try-catch (this call should never throw an authentication exception).

Reply:
Done, thx :)